components/layout/blog-view.hbs

```diff
@@ -0,0 +1,226 @@
+<div class="section--bg-white blog-feed">
+    <a href="#" class="content-block--clickable blog-content">
+        <div class="grid-col-content__two featured">
+            <div class="content-block">
```

Comment:
This feels... weird? On line 1 you've declared "this is a 'content block' that's modified as 'clickable'." Then this (and below, like line 9) says 'I'm redeclaring the stuff inside as a regular 'content-block.'" It feels very anti-patterny. A content block should be all the content, not sections inside of it. That would make them block__element level items.